po/fr.po

```diff
@@ -309,11 +309,11 @@ msgstr ""
 
 #: src/Views/PartitioningView.vala:60
 msgid "It is also recommended to select a <b>Swap</b> partition."
-msgstr ""
+msgstr "Il est également recommandé de sélectionner une partition d'<b>échange</b>."
 
 #: src/Views/PartitioningView.vala:89
 msgid "Modify Partitions…"
-msgstr ""
+msgstr "Modifier les partitions"
```

Comment:
```suggestion
msgstr "Modifier les partitions…"
```

`…` implies users that there needs additional user interaction after choosing this action, so you shouldn't omit it in your translations too.